### PR TITLE
chore: reorganize storybook

### PR DESCRIPTION
**Pull request by aslilac**
Screenshot 2023-10-09 at 11 52 39 AM

mostly just reorganizing pages stories. hoping it'll be a lot easier to find stuff now!

### CODE REVIEW

**Review by BrunoQuaresma — APPROVED** [review]
I don't have an opinion about how we organize things on storybook but I found this an improvement so I'm approving it 👍

**Comment by BrunoQuaresma** [comment]
PS: Just check and approve the changes on Chromatic 😆

File: site/src/components/Dashboard/Navbar/UserDropdown/UserDropdown.stories.tsx [hunk 1]
```diff
@@ -13,4 +13,6 @@ const meta: Meta<typeof UserDropdown> = {
 export default meta;
 type Story = StoryObj<typeof UserDropdown>;
 
-export const Example: Story = {};
+const Example: Story = {};
+
+export { Example as UserDropdown };
```
Comment (Parkreiner) [inline site/src/components/Dashboard/Navbar/UserDropdown/UserDropdown.stories.tsx:18]: Sorry, could you explain what's going on with this line? I feel like there's something Storybook-specific that I'm missing.

From what I can tell, the code is:
1. Exporting an anonymous/unnamed object
2. Passing in `Example` via object shorthand
3. It's also using the `as` keyword, but I'm not completely sure if this is the TS type-assertion version, or the JS module exports version

My guess is that this is just a named export that exports `Example` with the type of `UserDropdown`?

**Review by buenos-nachos — APPROVED** [review]
This looks really nice!

File: site/src/components/Dashboard/Navbar/UserDropdown/UserDropdown.stories.tsx (hunk 1 above)
Comment (aslilac) [reply site/src/components/Dashboard/Navbar/UserDropdown/UserDropdown.stories.tsx:18]: so there's no object literal here. it's analogous to `import { SomeThing as SomeThingElse } from "...";`.

Comment (Parkreiner) [reply site/src/components/Dashboard/Navbar/UserDropdown/UserDropdown.stories.tsx:18]: Okay, cool. That makes sense. Thanks for explaining!

**Pull request merged**